recipes/referenceseeker/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "1.2.0" %}
+{% set sha256 = "89ee028d45d1477f56e8d38ed2de53a1d706913c8fb5e57f1c0d12672c6216e2" %}
+
+package:
+  name: referenceseeker
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/oschwengers/referenceseeker/archive/v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - referenceseeker = referenceseeker.referenceseeker:main
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - biopython >=1.71
+    - mash >=2.1.0
+    - mummer4 >=4.0.0beta2
+
+test:
+  import:
+    - Bio.SeqIO
+  commands:
+    - referenceseeker --help
+    - mash dist -h
+    - nucmer -V
```

Comment:
There tests isn't required, it's the work of mash and mummer4 recipes.

Reply:
So true... I've seen this elsewhere and thought a double check wouldn't do any harm. 
Are there any best practices? Is it ok to keep it or shall I remove these tests?

Reply:
I have no clear opinion about it.